### PR TITLE
Reverting back to default SQLite DB

### PR DESCRIPTION
**Pull request by simon-dube**

### CODE REVIEW

File: server/config.py [hunk 1]
```diff
@@ -7,11 +7,15 @@
...
 
+DEFAULT_PROD_DB_URI = os.path.join(basedir, 'database/app.db')
+SQLITE_DEFAULT_PROD_DB_URI = 'sqlite:///{}'.format(DEFAULT_PROD_DB_URI)
+
 
 class Config(object):
     TESTING = False
     DEBUG = False
-    SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URI')
+    SQLALCHEMY_DATABASE_URI = (os.environ.get('DATABASE_URL')
```
Comment (louis-ver) [inline server/config.py]: Why did you change this from `URI` to `URL`?

**Review by louis-ver — APPROVED** [review]
Great stuff!

File: server/config.py (hunk 1 above)
Comment (simon-dube) [reply server/config.py]: Oups, I copy-pasted the line from an older version of the code to make sure not to make any mistake. It appears like we had not changed the variable name back then.

Comment (simon-dube) [reply server/config.py]: The README still had `DATABASE_URL`. I also changed it to reflect this change.
